gensim/similarities/docsim.py

```diff
@@ -629,7 +629,10 @@ def get_similarities(self, query):
         result = []
         for qidx in range(n_queries):
             # Compute similarity for each query.
-            qresult = [self.w2v_model.wmdistance(document, query[qidx]) for document in self.corpus]
+            try:
+                qresult = [self.w2v_model.wmdistance(document, query[qidx]) for document in self.corpus]
```

Comment:
What's a reason to check for both  `` and `.wv` ?

Reply:
Reverting back this change.